docs/configuration.md

```diff
@@ -450,8 +452,20 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.jars.ivy</code></td>
   <td></td>
   <td>
-    Comma-separated list of additional remote repositories to search for the coordinates given
```

Comment:
Yes, this is wrong documentation. At the time of the first PR we didn't document this, because we were unsure how popular the feature would be and we didn't want to confuse anyone with too many configurations. I think the time has come to add all this new functionality :)